util/tracing/trace_to_perfetto.py

```diff
@@ -41,13 +52,35 @@ def parse_arguments():
         metavar="<inputs>",
         type=argparse.FileType("rb"),
         nargs="+",
-        help="Input performance metric dumps",
+        required=True,
+        help='Input performance metric dumps ("*.trace.json")',
+    )
+    parser.add_argument(
+        "--traces",
+        metavar="<trace>",
+        nargs="+",
+        required=True,
+        help='Simulation traces to process ("*.dasm")',
     )
     parser.add_argument(
-        "--traces", metavar="<trace>", nargs="*", help="Simulation traces to process"
+        "--addr2line",
+        default="llvm-addr2line",
+        help="llvm-addr2line from quidditch toolchain",
+    )
+
+    # Only allow SNAX accelerators for now
+    snax_accelerators = []
+    for accelerator, acc_class in AcceleratorRegistry().registered_accelerators.items():
+        if issubclass(acc_class, SNAXAccelerator):
+            snax_accelerators.append(accelerator)
+
+    parser.add_argument(
+        "--accelerator",
+        choices=snax_accelerators,
+        default=None,
+        help="SNAX accelerator for SNAX Event Annotator",
```

Comment:
does this actually do something?

Reply:
This code makes sure that you can only call the argumentparser with SNAX-like accelerators